# Spatially resolved transcriptomics - introduction

In single-cell biology, a classical visualization of bulk RNA sequencing is a smoothie which resembles a diverse mixture of different cells all stired together. In single-cell RNA sequencing (scRNA-seq) cells are dissociated form the tissue and are analyzed based on their genes expression profile. So, scRNA-seq can be visualized by digging into a fruit salat. 

If we stick to the fruit motto, spatially resolved transcriptomic methods can be seen as a fruit tart. The exact position of every strawberry, banana and others is known and we can inspect the relationship between each piece. Spatial single-cell transcriptomics is becoming increasingly important and was announced Method of the Year in 2021 by Nature{cite}`marx_method_2021`. It measures transcriptomic information while spatial information is preserved. Spatial transciptomic data can be generated in multiple ways by measuring the RNA molecules by in situ sequencing, in situ hybridization or spatial barcoding. Each method aims to recover the original spatial coordinates. 

Adding the spatial information adds various new analysis and visualization opportunities and challenges which opens the field for new novel methods. {cite}`10.3389/fgene.2021.785290` provided a complete review of current analysis steps and methods for spatial transcriptomics data. We will summarize their findings in this chapter and show an examplary analysis workflow for for targetted and spot-based spatial transcriptomics data.

